doc/build/orm/declarative_styles.rst

```diff
@@ -450,7 +450,7 @@ A mapping using ``@attr.s``, in conjunction with imperative table::
 
```

Comment:
A couple of lines above it says

> A mapping using ``@attr.s``, in conjunction with imperative table::

That should be updated too. Also I think we may mention in that like that using the old style `attr.s` and `attr.ib` is also supported